dijit/1.11/dijit.d.ts

```diff
@@ -225,9 +219,7 @@ declare namespace dijit {
 	}
 
 	/* dijit/_FocusMixin */
-	interface _FocusMixin { }
-
-	interface _WidgetBase extends dojo.Stateful, Destroyable {
+	interface _FocusMixin extends _WidgetBase {
```

Comment:
According to https://github.com/dojo/dijit/blob/master/_FocusMixin.js#L57, `_FocusMixin` does not inherit from `_WidgetBase`. When the file is required, it augments `_WidgetBase` with a bunch of other properties, but if I tried to extend from`_FocusMixin` and expected all the properties from `_WidgetBase` to be there it would not work as I expect it to.

I'm not really sure how to express this stuff properly from the definition file.